src/main/java/racingcar/controller/Controller.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.controller;
+
+import racingcar.model.Cars;
+import racingcar.model.Race;
+import racingcar.view.Input;
+import racingcar.view.Output;
+
+public class Controller {
+    public void startGame() {
+        Input input = new Input();
+        String carNames = input.requestCarNames();
+        String raceRounds = input.requestRaceRounds();
+
+        Cars cars = new Cars();
+        cars.createRoundStatus(carNames);
```

Comment:
_createRoundStatus()_ 기능을 생성자 안에 넣어서 좀더 간결하게 작성할 수 있을 것 같아요. 

이름으로 의도를 드러내고 싶다면 정적 팩토리 메서드를 활용하면 좋겠다는 생각이 들었습니다. 

```suggestion
        Cars cars = Cars.createRoundStatus(carNames);
```

Reply:
네 생성자 내에 배치할 생각을 하지 못했네요! 조언 감사합니다!